src/config/2019.json

```
@@ -2,7 +2,7 @@
   "settings": [
     {
       "is_live": true,
-      "supported_languages": ["en","es","fr","ja"],
+      "supported_languages": ["en","es","fr","ja","ko"],
```

```suggestion
      "supported_languages": ["en","es","fr","ja"],
```

Once it's in this config, it will be live on the next release so can we keep it out of this config for now? At least until we have a chapter or two to launch.